pubtools/_pulp/tasks/garbage_collect.py

```diff
@@ -31,7 +32,19 @@ def add_args(self):
             default=5,
         )
 
+        self.parser.add_argument(
+            "--arc-threshold",
+            help="delete all-rpm-content older than this many days",
+            type=int,
+            default=180,
```

Comment:
Hmm, I think we should make this default a fair bit more aggressive.  How about 30?

An older variant of the cleanup code, rhsm-pulp-clean-rpms, used a default of 180 and it's probably been carried over from there.  However, the old code worked differently.  It used the timestamp of "file arrived in pulp" while the new code uses a different timestamp, cdn_published, meaning "file was published from Pulp to CDN".  I think it's fine to use that timestamp, but it means this can be quite more aggressive as we already know that the file must have been published from Pulp to CDN by the time we're cleaning it up.